querybook/webapp/components/QueryComposer/QueryComposerExecution.tsx

```diff
@@ -61,9 +63,11 @@ export const QueryComposerExecution: React.FunctionComponent<IProps> = ({
             </Level>
             <QueryExecution
                 id={id}
+                isAdhoc={true}
                 onSamplingInfoClick={onSamplingInfoClick}
                 hasSamplingTables={hasSamplingTables}
                 sampleRate={sampleRate}
+                changeCellContext={onUpdateQuery}
```

Comment:
i think this is a good indication that we should rename changeCellContext to onUpdateQuery since the param is no longer applicable